repositories/AnswerRepository.js

```diff
@@ -1,21 +1,25 @@
 const { head, invert, map } = require("lodash/fp");
 const Answer = require("../db/Answer");
 const mapFields = require("../utils/mapFields");
-
 const mapping = { QuestionPageId: "questionPageId" };
 const fromDb = mapFields(mapping);
 const toDb = mapFields(invert(mapping));
+const { STATE_ACTIVE, STATE_DELETED } = require("../constants/recordStates");
 
 module.exports.findAll = function findAll(
   where,
+  whereNot = { state: STATE_DELETED },
```

Comment:
This should be express in the positive (`where = {state : STATE_ACTIVE }`, or `where = { isDeleted : false }`)